docs/src/AlgebraicGeometry/AffineSchemes.md

```diff
@@ -6,56 +6,229 @@ CurrentModule = Oscar
 using Oscar
 ```
 
-# General schemes
+# General schemes and their interfaces
 
 Arbitrary schemes over a base ring ``\mathbb k`` which are given by means 
```

Comment:
Don't you want to mention somewhere that every ring here should at least be commutative with unity?

Reply:
I assumed this was understood. Has anyone ever seen schemes over non-commutative rings? 

Ok, there are the non-commutative C*-algebras, but that is hardly ever called scheme. 

Anyway: I think you are right. I will add that.